lucien-ios/LoginScreen/LoginViewModel.swift

```diff
@@ -49,4 +49,74 @@ final class LoginViewModel {
             }
         }
     }
+
+    // TODO: Replace completion (Error?) with completion (Comic?) when Comic model is complete.
+    func addComicBook(comicTitle: String,
+                      storyTitle: String,
+                      volume: String?,
+                      issueNumber: String?,
+                      publisher: String?,
+                      releaseYear: String?,
+                      comicPhotoURL: String?,
+                      returnDate: String?,
+                      condition: String?,
+                      genre: String?,
+                      completion: @escaping (Error?) -> Void) {
+
+        lucienAPIClient.addComicBook(comicTitle: comicTitle,
```

Comment:
All parameters here should be on a new line, indented by one from `lucienAPIClient`